SU-SWS/linky_clicky
# Add step definition to check if a checkbox is disabled

# READY FOR REVIEW

# Summary
- Adds a step definition to check if a checkbox is disabled

# Needed By (Date)
- Whenevah

# Criticality
- How critical is this PR on a 1-10 scale? 2

# Steps to Test

1. Set up a local site
2. Add `paranoia` and [this patch](https://www.drupal.org/files/issues/paranoia-block_additional_forms-2313945-22.patch)
3. Enable Paranoia
4. Run this Scenario:

```
  @api @javascript
  Scenario: Views Bulk Operations Chainsaw Mode
    Given I am logged in as a user with the "administrator" role
    And the "views_bulk_operations" module is enabled
    And I am on "admin/structure/views/view/admin_views_node"
    When I click "Bulk operations: Content"
    And I wait for AJAX to finish
    Then the "Execute arbitrary PHP script" checkbox should be disabled
    And the "Delete item" checkbox should be disabled
```

5. The "Execute arbitrary PHP script" step should **pass**
6. The "Delete item" step should **fail**
7. Confirm in the UI that "Delete item" is **enabled** and "Execute arbitrary PHP script" is **disabled**

# Affected Projects or Products
- ACSF

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/SITES-320
- Anyone who should be notified? ( @kbrownell )
